tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -296,9 +296,8 @@ def initial_mirror_config(self, mirror2, poolname, imagename, **kw):
             p.spawn(mirror2.create_pool, poolname=poolname)
 
         self.create_image(imagespec=imagespec, size=imagesize)
-        if kw.get("image_feature"):
-            image_feature = kw.get("image_feature")
-            self.image_feature_enable(imagespec=imagespec, image_feature=image_feature)
+        if kw.get("mode") == "pool":
```

Comment:
i don't see much difference to existing I feel old code also does the same 
what about `mode` as `image` and images are not enabled the required features  for mirroring?

Reply:
Added check for mirrormode, it will execute the image_feature_enable when either mode is "pool" or mirror mode is "journal".